### PR TITLE
docs(admin-api) securing the admin api

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: app/docs/0.10.x/admin-api.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ keep the configuration consistent across all nodes.
 - `8001` is the default port on which the Admin API listens.
 - `8444` is the default port for HTTPS traffic to the Admin API.
 
+This API is designed for internal use and provides full control over Kong, so
+care should be taken when setting up Kong environments to avoid undue public
+exposure of this API. See [this document][secure-admin-api] for a discussion
+on methods to secure the Admin API.
```
Comment (coopr) [inline app/docs/0.10.x/admin-api.md]: "of methods to..."

File: app/docs/0.10.x/secure-admin-api.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+---
+title: Securing the Admin API
+---
+
+# Overview
+
+Kong's Admin API provides a RESTful interface for administration and
+configuration of APIs, plugins, consumers, and credentials. Because this
+API allows full control of Kong, it is important to secure this API against
+unwanted access. This document touches on a few approaches to take to secure
```
Comment (coopr) [inline app/docs/0.10.x/secure-admin-api.md]: "...touches on a few approaches to take to secure..." can be shortened, and Colloquialism removed, by "...describes a few approaches to secure..."

File: app/docs/0.10.x/secure-admin-api.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  - [Reduce Listening Footprint](#reduce-listening-footprint)
+  - [Layer 3/4 Network Controls](#layer-34-network-controls)
+- [Kong API Loopback](#kong-api-loopback)
+- [Custom Nginx Configuration](#custom-nginx-configuration)
+
+## Network Layer Access Restrictions
+
+### Reduce Listening Footprint
+
+By default, Kong will accept requests for both the public-facing entrypoint, and
```
Comment (coopr) [inline app/docs/0.10.x/secure-admin-api.md]: Is "entrypoint" a term we use elsewhere in our docs? Or is there some other term, that means that, that we use instead? Or is this the more-correct term, and we should be changing it elsewhere?

File: app/docs/0.10.x/secure-admin-api.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+[Back to TOC](#table-of-contents)
+
+### Layer 3/4 Network Controls
+
+In cases where the Admin API must be exposed beyond a localhost interface,
+network security best practices dictate that network-layer access be restricted
+as much as possible. Consider an environment in which Kong listens on a private
+network interface, but should only be accessed by a small subset of an IP range.
+In such a case, host-based firewalls (e.g. iptables) are useful in limiting
+input traffic ranges. Consider the following example:
```
Comment (coopr) [inline app/docs/0.10.x/secure-admin-api.md]: "Consider..." starts two sentences in the same paragraph - consider changing one of them to something other than "Consider"

File: app/docs/0.10.x/secure-admin-api.md (hunk 3 above)
Comment (p0pr0ck5) [inline app/docs/0.10.x/secure-admin-api.md]:
```
$ git grep entrypoint
app/docs/0.10.x/admin-api.md:The entrypoint to the clustering API functionalities. Shows the total number of events that have been handled by the current node, and their types. The types starting with `ENTITY_` are events for database entities, while the types that start with `MEMBER-` are events of the cluster and its members.
app/docs/0.10.x/configuration.md:public-facing entrypoint of Kong, to which your consumers will make
app/docs/0.10.x/configuration.md:Address and port on which Kong will expose an entrypoint to the Admin API.
app/docs/0.10.x/secure-admin-api.md:By default, Kong will accept requests for both the public-facing entrypoint, and
app/docs/0.9.x/admin-api.md:The entrypoint to the clustering API functionalities. Shows the total number of events that have been handled by the current node, and their types. The types starting with `ENTITY_` are events for database entities, while the types that start with `MEMBER-` are events of the cluster and its members.
app/docs/0.9.x/configuration.md:public-facing entrypoint of Kong, to which your consumers will make
app/docs/0.9.x/configuration.md:Address and port on which Kong will expose an entrypoint to the Admin API```

**Comment by p0pr0ck5** [comment]
Thanks, adjusted as per comments, with the exception of 'entrypoint', which we do use throughout the docs. Is there a place in the new doc this term should be used that I missed?

File: app/docs/0.10.x/secure-admin-api.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+title: Securing the Admin API
+---
+
+# Overview
+
+Kong's Admin API provides a RESTful interface for administration and
+configuration of APIs, plugins, consumers, and credentials. Because this
+API allows full control of Kong, it is important to secure this API against
+unwanted access. This document describes a few approaches to take to secure
+the Admin API.
```
Comment (Tieske) [inline app/docs/0.10.x/secure-admin-api.md]: "approaches to take to secure" not native speaking, but looks a bit awkward  to me? please ignore this comment if not

Comment (p0pr0ck5) [inline app/docs/0.10.x/secure-admin-api.md]: Yeah, this was a bit awkward. Reworked.

**Pull request merged**